dist/error-codes.md

```diff
@@ -74,6 +74,7 @@ Critical errors indicate a problem with the Magento Commerce Cloud project confi
 | 132 |  | Can not connect to the Elasticsearch service | Check that credentials for elasticsearch are correct and service is running |
 | 133 | validate-config | Remove Magento Braintree module configuration which is no longer supported in Magento 2.4 and later versions. | Support for the Braintree module is no longer included with Magento 2.4.0 and later. Remove the CONFIG__STORES__DEFAULT__PAYMENT__BRAINTREE__CHANNEL variable from the variables section of the .magento.app.yaml file. For Braintree support, use an official Braintree Payments extension from the Magento Marketplace instead. |
 | 134 | validate-config | Magento 2.4.0 requires Elasticsearch service to be installed | Install Elasticsearch service |
```

Comment:
Actually, error name `DEPLOY_WRONG_SEARCH_ENGINE` does not fit to this description. I would rename it and set `DEPLOY_WRONG_SEARCH_ENGINE` for new error, which you implemented here.
* as these name for internal use only, so we are good to change them

Reply:
hm, those names are constants...
from other side, we did not mark this class as @api, so..

Reply:
Agree, renamed

Reply:
Technically this is a small BIC change, but as you said we didn't mark this class as @api